### PR TITLE
feat: add formik example

### LINKED ISSUE #113: Formik example
**Describe the example you'd like**

Basic example of universal forms with https://jaredpalmer.com/formik

### PR DESCRIPTION
**Pull request by amandeepmittal**
Fixes #113.

I have made a small example of how to create and use Formik to create forms in Expo apps, and use validation (using yup) on login and signup screens. The example contains three screens:

- Login (to allow user login the app using credentials)
- Signup (to allow user create a new user account)
- Home (a basic screen with logout button)

I'm using personal Firebase project API keys. If that is not an optimal way, please let me know and I'll make amendments.

If there is any other change required, please let me know and I will make changes accordingly in the example.

### CODE REVIEW

**Comment by EvanBacon** [comment]
The examples all need to be super minimal since they are used as templates in CRNA. A formik example should probably not have any firebase in it.

This example is still pretty useful (I'd use it), perhaps you should make a personal repo informing people to use it in the README like:
```
npx crna -t https://github.com/amandeepmittal/expo-firebase-formik-template
```

File: with-formik/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (EvanBacon) [inline with-formik/package.json:1]: Nice job getting the pkg json right 👍

**Comment by amandeepmittal** [comment]
Thanks for going through it and reviewing it, Evan. 

For the minimal example, should I just include one screen (login component) and basic Formik with yup?

Let me get back to you with a more minimal example.

File: with-formik/package.json (hunk 1 above)
Comment (amandeepmittal) [reply with-formik/package.json:1]: The credit goes to you for that. The documentation is clear and expo-cli helps where in doubt :)

**Comment by amandeepmittal** [comment]
I have changed the templates to more Formik centric with minimal validation using yup and only one screen component. Let me know if this is okay?

**Comment by amandeepmittal** [comment]
Thanks, Evan :)

**Pull request merged**